ubuntu/dbfsfuse/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM databricksruntime/python2.7-virtualenv:latest
```

Comment:
What would this look like for Python 3? Is there an analogous virtualenv and is this how we should standardize?